### PR TITLE
Add records for staging.elinks.judiciary.uk

### PR DESCRIPTION
**Pull request by andyrogers1973**
Updated/added the following records:

SPF
TXT name: [staging.elinks.judiciary.uk](http://staging.elinks.judiciary.uk/)
SPF Value: v=spf1 include:[spf.protection.outlook.com](http://spf.protection.outlook.com/) -all
TTL: 3600

DKIM
CNAME record name: selector1-azurecomm-prod-net._domainkey
DKIM value: selector1-azurecomm-prod-net._[domainkey.azurecomm.net](http://domainkey.azurecomm.net/)
TTL: 3600

DKIM2
CNAME record name: selector2-azurecomm-prod-net._domainkey
DKIM value: selector2-azurecomm-prod-net._[domainkey.azurecomm.net](http://domainkey.azurecomm.net/)
TTL: 3600

### CODE REVIEW

**Review by AntonyBishop — CHANGES_REQUESTED** [review]
Need to check the full subdomain of the CNAMES. The request suggests these all relate to `staging`, but thiose records will not create staging subdomains.

Need to check is the intent is for these to be `selector1-azurecomm-prod-net._domainkey.staging.elinks.judiciary.uk` or `selector1-azurecomm-prod-net._domainkey.judiciary.uk`. 

Applies to both CNAMES.

File: hostedzones/judiciary.uk.yaml [hunk 1]
```diff
@@ -375,9 +375,17 @@ s2._domainkey.staging.hr:
 selector1._domainkey:
   type: CNAME
   value: selector1-judiciary-uk._domainkey.justiceuk.onmicrosoft.com.
+selector1-azurecomm-prod-net._domainkey:
+  type: CNAME
+  value: selector1-azurecomm-prod-net._domainkey.azurecomm.net
+  ttl: 3600
```
Comment (AntonyBishop) [inline hostedzones/judiciary.uk.yaml:381]: `ttl` needs to come before `type`. This is the cause of the sync failed check.

File: hostedzones/judiciary.uk.yaml [hunk 2]
```diff
@@ -375,9 +375,17 @@ s2._domainkey.staging.hr:
...
 selector2._domainkey:
   type: CNAME
   value: selector2-judiciary-uk._domainkey.justiceuk.onmicrosoft.com.
+selector2-azurecomm-prod-net._domainkey:
+  type: CNAME
+  value: selector2-azurecomm-prod-net._domainkey.azurecomm.net
+  ttl: 3600
```
Comment (AntonyBishop) [inline hostedzones/judiciary.uk.yaml:388]: `ttl` needs to come before `type`. This is the cause of the sync failed check.

File: hostedzones/judiciary.uk.yaml (hunk 1 above)
Comment (andyrogers1973) [reply hostedzones/judiciary.uk.yaml:381]: Thanks, will update

File: hostedzones/judiciary.uk.yaml (hunk 2 above)
Comment (andyrogers1973) [reply hostedzones/judiciary.uk.yaml:388]: Ditto

**Review by AntonyBishop — APPROVED** [review]

**Pull request merged**